cmd/porter/bundle.go

```diff
@@ -63,7 +63,7 @@ func buildBundleBuildCommand(p *porter.Porter) *cobra.Command {
 			return opts.Validate(p)
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return p.Build(opts)
+			return p.Build(cmd.Context(), opts)
```

Comment:
FYI, short of using global package variables (which would cause problems for anyone using porter as a library) we are going to just have to pass context around everywhere going forward.

It's useful for other things, like cancel support, so it's not the worst, just ugly. 😀